src/main/java/appeng/util/item/AEFluidStack.java

```diff
@@ -347,7 +336,7 @@ public Fluid getFluid()
 	public ItemStack asItemStackRepresentation()
 	{
 		// TODO: fluids, how do they even work?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)